integration/deploy/compose_test.go

```diff
@@ -0,0 +1,308 @@
+//go:build integration
+// +build integration
+
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deploy
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/okteto/okteto/integration"
+	"github.com/okteto/okteto/integration/commands"
+	"github.com/okteto/okteto/pkg/registry"
+	"github.com/stretchr/testify/require"
+	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
+)
+
+const (
+	composeTemplate = `services:
+  app:
+    build: app
+    entrypoint: echo -n $RABBITMQ_PASS > var.html && python -m http.server 8080
```

Comment:
Where is the `RABBITMQ_PASS` environment variable defined? In out tests or in the Okteto instance where the tests is running?

If it is defined in the Okteto instance, It is sensible to fail if anyone change or remove that (any Okteto admin can do it). I think we should document all the external dependencies of that kind we have and be strict on updating it. 

Other possibility is to have a standard for this case and define these variable as `E2E_TEST_RABBITMQ_PASS`. Including the prefix you give some context to indicate that those variable are being used for e2e tests

Reply:
I need to create a document on how to run e2e tests locally for defining all these scenarios. `RABBITMQ_PASS` is a user level secret (I'm thinking now that testing secret injection should be run on the okteto folder, so we can run all the other tests on vanilla clusters). I haven't change the name because it's already defined in `cindylopez` user (the one that we use to run e2e tests)

Reply:
But we can create the new secret in `cindylopez` and start using it.

And I would try to avoid to rely on a specific user (or info from the user) to run these tests. That makes tests error prone. An option could be to define a local `.env` in the test with the right value. In that way, you can also test secret injection and it would be a valid scenario for vanilla clusters

Reply:
+1 we should allow these to be run on any cluster with any user.